opencog/cython/opencog/atom.pyx

```diff
@@ -10,7 +10,7 @@ cdef class Atom(object):
         del self.handle
 
     def value(self):
-        return void_from_cptr(self.handle)
+        return self.handle.value()
```

Comment:
Oh, I think I get it -- you want it to be consistent with what the c++ code calls the 'value, which, I guess is probably returning the hash, right now.  I'm thinking that it might be best to remove 'value' from both the c++ and python API -- there is not that much use for it, except for printing, and some sanity-checking.